rundeckapp/grails-app/services/rundeck/services/LoggingService.groovy

```diff
@@ -63,7 +63,7 @@ class LoggingService implements ExecutionFileProducer {
     def grailsLinkGenerator
 
     public boolean isLocalFileStorageEnabled() {
-        boolean fileDisabled = grailsApplication.config?.rundeck?.execution?.logs?.localFileStorageEnabled in ['false', false]
+        boolean fileDisabled = grailsApplication.config?.getProperty("rundeck.execution.logs.localFileStorageEnabled",String.class) in ['false', false]
```

Comment:
probably configurationService should be used instead of `grailsApplication.config` in any place that loads a config value starting with `rundeck.`